runConfigurations/Cromwell_server.run.xml

```diff
@@ -8,6 +8,8 @@
       <env name="CROMWELL_BUILD_RESOURCES_DIRECTORY" value="target/ci/resources" />
       <env name="CROMWELL_BUILD_PAPI_JSON_FILE" value="target/ci/resources/cromwell-centaur-service-account.json" />
       <env name="CROMWELL_BUILD_CENTAUR_READ_LINES_LIMIT" value="128000" />
+      <env name="CROMWELL_BUILD_PAPI_DOCKER_IMAGE_DRS" value="broadinstitute/cromwell-drs-localizer:develop" />
+      <env name="CROMWELL_BUILD_CENTAUR_256_BITS_KEY" value="AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=" />
```

Comment:
![aaaaa](https://user-images.githubusercontent.com/19228696/139491994-830c6f41-c4c9-43a4-87e9-df11f09b3fb3.gif)

But seriously, what does this mean?

Reply:
There's a 64 bit key used to encrypt any workflow options that are specified in config as being sensitive. There are actually [quite a few of these fields](https://github.com/broadinstitute/cromwell/blob/935ecc9cefa45af36fb537c065fe5ca45c6afb63/src/ci/resources/build_application.inc.conf#L6) and whenever we try to run a Centaur test that submits a workflow with one of these options it would [fail with a cryptic error](https://broadinstitute.slack.com/archives/GHYJZ2ZE0/p1630705913106400) about the key being too long because Cromwell was trying to use the "not defined" error message as a key.

Reply:
Fun